src/status_im2/utils/simple_message_resolver.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im2.utils.simple-message-resolver
+  (:require [utils.re-frame :as rf]))
+
+(defn resolve-message
```

Comment:
@rasom, would you mind adding a unit test for this function? I actually think you had created one from another PR, no? Or at least it was very similar to this implementation.

Reply:
well it is already in this PR, I just renamed test's ns

Reply:
You're right about that! Sorry ;)